docs/src/main/sphinx/sql/set-session.rst

```diff
@@ -15,28 +15,41 @@ Description
 
 Set a session property value or a catalog session property.
 
+.. _session-properties-definition:
 
-Connectors can provide session properties. They are set separately for each
-catalog by prefixing them with the catalog name. There is no sharing of
-properties across catalogs, even if the catalogs use the same connector.
+Session properties
+------------------
+
+A session property is a :doc:`configuration property </admin/properties>` that
+can be temporarily modified by a user for the duration of their current
+connection to the Trino cluster. Many configuration properties have a
+corresponding session property that accepts the same values as the config
+property.
+
+A catalog session property is a connector-defined session property that is set
+on a per-catalog basis. These properties are set separately for each catalog
+by including the catalog name as a prefix.
```

Comment:
```suggestion
A catalog session property is a connector-defined session property that can be set
on a per-catalog basis. These properties can be set separately for each catalog
by including the catalog name as a prefix.
```

(might be good to have an example of what the prefixing looks like - note: it isn't catalog-property or catalogproperty but catalog.property)